docs/docker.md

```diff
@@ -0,0 +1,65 @@
+## Setting up Docker
+
+Bison includes a `docker-compose.yml`, `Dockerfile` and `.dockerignore`
+
+The configuration is to run a postgres database locally. 
+
+If you are interested in running the entire app, please see the examples below.
+
+## Running the Database
+
+To run postgres with docker, please add the following information to your `env` & `env.local` file.
+```
+DATABASE_URL="postgresql://dev:dev@localhost:5432/dev?schema=public"
+```
+Run `docker-compose up -d` in your terminal. If you prefer not to run in detached mode, you can remove the `-d` flag.
+
+To shut down docker, run `docker-compose down`
+
+## Using NextJS & Postgres with Docker
+
+The following configuration will help you run NextJS & Postgres. 
+
+Make sure `@prisma/client` is on version: `3.13.0`,
```

Comment:
I will eliminate this. I was writing this when working on `node:alpine` Prisma does not work well on < 3.13.0

The error was related to running `node:alpine`. It will not apply if we are using `node:lts` 

When following NextJS docs they are using node:alpine (https://github.com/vercel/next.js/blob/canary/examples/with-docker/Dockerfile). Running Prisma and alpine has some issues and upgrading made it work: 

https://github.com/prisma/prisma/discussions/13218#discussioncomment-2700752